Frontend/ui-driver/src/Resource/Localizable/EN.purs

```diff
@@ -655,15 +655,15 @@ getEN stringKey =
         ENJOY_THESE_BENEFITS -> "Your joining benefits!"
         CHOOSE_YOUR_PLAN _ -> "Choose a Plan Now!"
         SKIP_FOR_NOW -> "Skip For Now"
-        SEVEN_DAY_FREE_TRIAL_ACTIVATED -> "7-Day FREE TRIAL Activated!"
-        TAKE_UNLIMITED_RIDES_FOR_THE_NEXT_SEVEN_DAYS -> "Take unlimited rides for the first 7 days"
+        SEVEN_DAY_FREE_TRIAL_ACTIVATED -> "{}-Day FREE TRIAL Activated!"
+        TAKE_UNLIMITED_RIDES_FOR_THE_NEXT_SEVEN_DAYS -> "Take unlimited rides for the first {} days"
         EVERY_RIDE_AT_ZERO_COMMISSION -> "Every ride at ZERO Commission!"
         EARN_UPTO_PER_DAY -> "Earn upto ₹{} a day!"
         HOW_THIS_WORKS -> "How this Works"
         SIGN_UP_FOR_AUTOPAY_BY_PAYING_JUST -> "Sign up for Autopay by paying just ₹1"
         GET_REMINDED_ABOUT_YOUR_PLAN_SETUP -> "Get reminded about your plan setup"
-        FREE_TRIAL_REMINDER -> "Day 5: FREE TRIAL reminder"
-        PLAN_STARTS -> "Day 7: Plan Starts"
+        FREE_TRIAL_REMINDER -> "Day {}: FREE TRIAL reminder"
+        PLAN_STARTS -> "Day {}: Plan Starts"
```

Comment:
why can't we just pass argument with constructor and use that parameter directly instead of handling regex in JS

Reply:
We can do but It takes so much time to add a single string